filters/encoding/filter.py

```diff
@@ -0,0 +1,15 @@
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+
+
+class TextEncodingFilter(SentenceOperation):
+    tasks = [e for e in TaskType]
+    languages = ["en", "de"]
```

Comment:
Wouldn't this work for "all" in addition to en and de?

Reply:
Sure! Didn't know, whether "all" is a valid setting.